Sources_v3/StreamChatUI/Utils/UIView+Extensions.swift

```diff
@@ -10,18 +10,18 @@ extension UIView {
         subview.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
             subview.leadingAnchor.constraint(equalTo: leadingAnchor, constant: insets.leading),
-            subview.trailingAnchor.constraint(equalTo: trailingAnchor, constant: insets.trailing),
+            subview.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -insets.trailing),
             subview.topAnchor.constraint(equalTo: topAnchor, constant: insets.top),
-            subview.bottomAnchor.constraint(equalTo: bottomAnchor, constant: insets.bottom)
+            subview.bottomAnchor.constraint(equalTo: bottomAnchor, constant: -insets.bottom)
         ])
     }
     
     func embedUsingSystemSpacing(
         _ subview: UIView,
         topInsetMultiplier: CGFloat = 1,
-        bottomInsetMultiplier: CGFloat = 1,
+        bottomInsetMultiplier: CGFloat = -1,
         leadingInsetMultiplier: CGFloat = 1,
-        trailingInsetMultiplier: CGFloat = 1
+        trailingInsetMultiplier: CGFloat = -1
```

Comment:
This helper doesn't work properly (take a look at https://github.com/GetStream/stream-chat-swift/pull/588/files#r523258367).

Reply:
I have no idea what's happening here, to be honest :) Why do we need negative multipliers? You just need to swap the order in which way you set up the constraints, don't you?
aka:
A.contraint(B, -1) == B.constraint(A, 1)

Maybe I'm wrong but I think that's how it should work :)

Reply:
The docs says: 
> receiver [= | ≥ | ≤] 'anchor' + 'multiplier' * system space

When we work with `trailing/bottom` anchors we specify the negative offset to make it behave as we expect. I realized we can provide `multiplier == -1` and have the correct inset but seems we can't.

Reply:
Removed this helper. We can use the `preservesSuperviewLayoutMargins` and subview the superviews anchors